runtime/src/main/kotlin/kotlin/Annotations.kt

```diff
@@ -148,4 +148,14 @@ public annotation class DslMarker
 @Retention(AnnotationRetention.BINARY)
 @MustBeDocumented
 @SinceKotlin("1.1")
-public annotation class PublishedApi
\ No newline at end of file
+public annotation class PublishedApi
+
+/**
+ * When applied to class marks the fact, that its instances could be copied by value with [toFrozen()]
+ * function by cloning object instance, and so when used in models could be safely cloned.
+ * Data classes are assumed to be shared by value by default.
+ */
+// TODO: how to properly mark as experimental?
```

Comment:
> how to properly mark as experimental?

Like this:

https://github.com/JetBrains/kotlin-native/blob/0f6859f29c43fc0f55430fbd6014d760400e066b/runtime/src/main/kotlin/kotlin/reflect/AssociatedObjects.kt#L19